README.md

```diff
@@ -0,0 +1,57 @@
+# Software Developers Global Summit 2020
+Annual conference for developers around the world.
+
+## Project Design:
+
+![screenshot](idesign/images/page-screenshot.PNG)
+
+This project was developed following the [CC Global Summit 2015](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) design by [Cindy Shin](https://www.behance.net/adagio07). It is a Microverse capstone project that I developed using bootstrap and a few more technologies (mentioned below). 
+
+Software developers Global Summit 2020 is a conference held at the end of each year. The site consist of three (Home, About, & Tickets) pages which do not have any functionality yet. The goal is to follow the given ([CC Global Summit 2015](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)) design, & personalise the content to anything relating to conferencing. You can see the project live demo (link provided below). Each of the pages on the site has got the mobile screen version (max-width ==> 768px) & desktop version (min-width ==> 768px).
+
+## Built With
+
+- CSS, HTML
+- Sass, Bootstrap 4
+- Stylelint validator
+
+## SetUp
+To get this project on your machine & contribute to the project, you can run <code>git clone https://github.com/den-cloud/Capstone-Project-1.git</code> on the command line and create a feature branch to your directory both locally and remotely.
+
+## Tools Used:
+
+- Visual Studio Code
+
+## Live Demo
+
```

Comment:
- [ ] You should also add a live demo for the project

Reply:
> * [x]  You should also add a live demo for the project

This issue is well addressed! Nice work! :clap: